Source/Renderer/RenderState.js

```diff
@@ -173,12 +182,12 @@ define([
             (!defined(viewport.width)) ? context.drawingBufferWidth : viewport.width,
             (!defined(viewport.height)) ? context.drawingBufferHeight : viewport.height) : undefined;
 
+        //>>includeStart('debug', pragmas.debug);
```

Comment:
Below is a `RuntimeError` so we do not want to optimize it out in release builds.